nest-forms-backend/src/nases/utils-services/tokens.nases.service.ts

```diff
@@ -292,24 +316,50 @@ export default class NasesUtilsService {
     return result; */
   }
 
-  private formatXmlToOneLine(xml: string): string {
-    const result = xml
-      .replaceAll(/\s+/g, ' ') // replace all multiple whitespaces with just one space
-      .replaceAll(/>\s+</g, '><') // delete all whitespaces between tags
-      .replaceAll(/\s*<\s*/g, '<') // delete all whitespaces (spaces) around tag marks
-      .replaceAll(/\s*>\s*/g, '>')
-      .replaceAll(/\s*\/>/g, '/>')
-      .replaceAll(/<\/\s*/g, '</')
-      .trim()
+  private async getFormMessage(
+    form: Forms,
+    isSigned: boolean,
+  ): Promise<string | object> {
+    let message: string | object | null = null
+    const parser = new Parser({ normalize: true, trim: true })
```

Comment:
I've just noticed those options, we definitely don't want them, the data are user-provided and these are modifications that we shouldn't do.

Reply:
I understand why do not want to use "normalize", but I do not know why we do not want to use `trim` option. Previously, the function `formatXmlToOneLine` performed trimming. If we don't utilize the `trim` option, newlines in user input may be preserved. (Newlines between <tags> are removed and not returned in Builder). Should we adjust our approach to only eliminate newlines and preserve blocks of whitespace, or should we opt to keep the newlines as well?

example input:
```xml
<root>
  < tag1   attribute="value1"/>
  <   tag2   
    attribute="value2">Content</tag2> 
  <tag3      >Another content    
       </  tag3>     
</root>
```

example output without trim:
```xml
<root><tag1 attribute="value1"/><tag2 attribute="value2">Content</tag2><tag3>Another content    
               </tag3></root>
```

Reply:
Yes, there's nothing wrong with preserving new lines, users type in content like this (e.g. in textareas) and we want to keep it as is. I would just remove parser options.